XIVComboExpanded/Combos/RDM.cs

```diff
@@ -426,8 +427,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
     {
         if (actionID == RDM.Embolden)
         {
-            if (level >= RDM.Levels.Manafication && IsCooldownUsable(RDM.Manafication) && !IsCooldownUsable(RDM.Embolden))
-                return RDM.Manafication;
+            if (level >= RDM.Levels.Manafication && !IsCooldownUsable(RDM.Embolden))
+            {
+                if (IsCooldownUsable(RDM.Manafication) && InCombat())
+                    return RDM.Manafication;
+                if (HasEffect(RDM.Buffs.ThornedFlourish))
+                    return RDM.ViceOfThorns;
+                if (HasEffect(RDM.Buffs.PrefulgenceReady) || HasEffect(RDM.Buffs.Manafication))
```

Comment:
This will cause issues for RDMs below Prefulgence level.  Also, why is the OR condition here at all?